_layouts/meltano_plugin.html

```diff
@@ -326,21 +295,47 @@ <h3 id="next-steps">Next steps</h3>
 
 <h2 id="capabilities">Capabilities</h2>
     
-{% if variant.capabilities.size == 0 %}
+{% if plugin.capabilities.size == 0 %}
 <p>
-  Check the
-  <a href="{{variant.repo}}#readme" target="_blank">README and code in the repository</a>
-  for more information on capabilities for this {{plugin_type}}.
+  <code>{{plugin.name}}</code> does not have any capabilities defined in its metadata.
+  Please considering adding them by making a PR to the
+  <a href="{{yaml_file}}">YAML file</a>
+  that defines the capabilities for this {{plugin_type}}.
 </p>
-{% else %} <ul> {% for capability in plugin.capabilities %} 
+
+{% else %} 
+
+  {% if plugin.capabilities.size < 3 %}
```

Comment:
For lots of taps, 2 capabilities are perfectly fine, and I don't think we should suggest to users that it's not. 

Since this is to catch taps that now have `state` and `catalog` as defaults even if we haven't verified that, I think we're better off removing the defaults or making it explicit with a property like `metadata_verified` (that could be `false` by default and set to `true` on the subset that are). We can then also clarify that this tap may not work out-of-the-box, and what to do if they hit an issue. It may not just be missing capabilities, but could also have incorrect ones (like `catalog` instead of `properties`).

I think it's important that people know from the plugin page whether they can expect the tap to work out of the box, or not. These paragraphs hint at that, but I don't think they set expectations sufficiently yet, or help a user address potential issues with taps with missing/broken metadata.

Reply:
@DouweM I understand the concerns, I just don't know how we're going to keep that up to date over the long run. A static `metadata_verified` will work for the short-term but is likely to be wrong as updates to the underlying package are created. We have https://github.com/meltano/hub/issues/219 which would give us a better indication of metadata versioning on a more automated basis. 

The original capabilities that I set when we first launched the Hub were `catalog` and `discover`. What do you think about softer language that only triggers when those 2 capabilities are set AND there are no settings?

Reply:
@DouweM take a look at the latest changes f78fe9048830759c804262e2fff6546a98fb752c

Reply:
@tayloramurphy I like https://github.com/meltano/hub/issues/219 and this workaround, thanks!